### PR TITLE
Feature/improve allocation by symbol

### PR DESCRIPTION
**Pull request by dtslvr**
I did some improvements, is it still working for you @I-Valchev?

### CODE REVIEW

File: apps/client/src/app/pages/portfolio/allocations/allocations-page.component.ts [hunk 1]
```diff
@@ -223,6 +218,15 @@ export class AllocationsPageComponent implements OnDestroy, OnInit {
               : this.portfolioDetails.holdings[symbol].value;
         }
       }
+
+      if (position.assetSubClass !== AssetClass.CASH) {
```
Comment (dtslvr) [inline apps/client/src/app/pages/portfolio/allocations/allocations-page.component.ts:222]: Ignore cash holding

File: apps/client/src/app/pages/portfolio/allocations/allocations-page.component.ts [hunk 2]
```diff
@@ -223,6 +218,15 @@ export class AllocationsPageComponent implements OnDestroy, OnInit {
...
+        // Prepare analysis data by symbols except for cash
+
+        this.symbols[symbol] = {
+          name: symbol,
+          value: aPeriod === 'original' ? position.investment : position.value
```
Comment (dtslvr) [inline apps/client/src/app/pages/portfolio/allocations/allocations-page.component.ts:227]: Respect period: "Initial" vs. "Current"

File: apps/client/src/app/pages/portfolio/allocations/allocations-page.html [hunk 3]
```diff
@@ -19,10 +19,9 @@ <h3 class="d-flex justify-content-center mb-3" i18n>Allocations</h3>
         <mat-card-content>
           <gf-portfolio-proportion-chart
             [baseCurrency]="user?.settings?.baseCurrency"
-            [isInPercent]="hasImpersonationId"
+            [isInPercent]="false"
```
Comment (dtslvr) [inline apps/client/src/app/pages/portfolio/allocations/allocations-page.html:22]: `hasImpersonationId` does not work here yet

**Comment by I-Valchev** [comment]
👍 Looks good!

**Pull request merged**